### PR TITLE
added clone to transaction structs

### PR DESCRIPTION
**Pull request by bengtlofgren**
## Describe your changes

Derives `Clone` and `Debug` for the light sdk structs.

## Indicate on which release or other PRs this topic is based on

v0.35.1

## Checklist before merging to `draft`
- [x] I have added a changelog
- [x] Git history is in acceptable state

### CODE REVIEW

File: crates/light_sdk/src/transaction/account.rs [hunk 1]
```diff
@@ -14,6 +14,8 @@ const TX_REVEAL_PK_WASM: &str = "tx_reveal_pk.wasm";
 const TX_UPDATE_ACCOUNT_WASM: &str = "tx_update_account.wasm";
 
 /// Transaction to initialize an established account
+#[derive(Debug, Clone)]
+
```
Comment (brentstone) [inline crates/light_sdk/src/transaction/account.rs:18]: remove this empty lines and everywhere else

**Comment by brentstone** [comment]
Take the scripts out of the changelog commit

**Comment by brentstone** [comment]
@grarco is this still needed?

**Comment by grarco** [comment]
Yes, my bad, I forgot to add this to the queue of the last draft branch. I'll rebase it and mark it as ready for draft

**Review by brentstone — APPROVED** [review]

**Pull request merged**